independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/ClientRequestContextImpl.java

```diff
@@ -225,12 +228,23 @@ public Annotation[] getEntityAnnotations() {
 
     @Override
     public OutputStream getEntityStream() {
-        throw new NotImplementedYet();
+        if (!hasEntity())
+            return null;
+
+        try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
+            ObjectOutputStream oos = new ObjectOutputStream(baos);
+            oos.writeObject(getEntity());
+            oos.close();
+            return baos;
+
+        } catch (IOException exception) {
+            return null;
+        }
     }
 
     @Override
     public void setEntityStream(OutputStream outputStream) {
-        throw new NotImplementedYet();
+        this.entityStream = outputStream;
```

Comment:
so it will be available only if someone sets if via filter, right? Wouldn't that be confusing?

Reply:
@michalszynkiewicz  I do not know where can I set that? Would you mind to guide me?